### ISSUE TITLE
accuracy of the CIFAR10 

### ISSUE THREAD

**Issue by blueardour** [zzzxxxttt/pytorch_DoReFaNet#3]
Hi, @zzzxxxttt . Thank you for the great project. I'm reproducing the experiment of training resnet20 on cifar10. However, I didn't obtain the same accuracy as your website reported.

For the full precision of resnet20 on cifar10, I got 91.79% as the best at epoch 75. Could I ask if  the number in your website was obtained exactly by the nearest commit of the code? From my experiment, the best accuracy was obtained at epoch 75, which indicated that we might not train 200 epochs or we could decrease to lower learn rate for given epochs.

Quantized version also gave a litter lower accuracy than yours. For example, w1a32 in my training gave 89.85% for the best.

Would you please give some advise?

**Comment by zzzxxxttt** [comment]
Hi @blueardour, the listed results are obtained by an earlier commit. I will rerun the experiments to see if there is something wrong with the latest commit, thanks for pointing out this issue!

**Comment by zzzxxxttt** [comment]
I found the problem, it's because the learning rate decrease too early, now it's fixed in the new commit. @blueardour

**Comment by blueardour** [comment]
hi, I got the expected accuracy with your advise. Thanks very much.
